docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,98 @@
+
+## Introduction
+
+Toolpad is a developer tool primarily for the admin applications use case. Building an admin panel (aka internal tools) can be intimidating because:
+
+- it is not focused on frontend
+- it is difficult to keep up with the fast evolving landscape of frameworks, libraries and best practices
+- lots of glue code is required to display the data
+- it requires connecting to multiple data sources
+- security of the underlying sensitive data needs to ensured at all times
+- it serves multiple stakeholders with varying needs
+
+Like any application, there are multiple points of failure that can lead to a sub-optimal user experience for end users (generally operations, finance, marketing teams). Since these don't go through rigorous quality assurance, failure chances are usually higher.
+
+MUI Toolpad aims to reduce the overhead associated with frontend development by providing visual editors and reducing the amount of boilerplate required to bring data on the page. With basic frontend knowledge you can build an app in just one sitting. On this page, we'll try to address your concerns by covering all aspects of building an admin panel. This shall help you evaluate if Toolpad is the right product for you.
+
+## TLDR
+
+- **Integrates well with your existing code**. You can reuse your database models, client libraries, secrets, bespoke components
+- Integrates well with your **development lifecycle.** Your project lives on your own file system. You can use your own IDE, version control system, deployment target, CLI tools
+- Comes with a **Drag and drop builder** to simplify UI building
+- Sets **Healthy UI constraints** that prevent common pitfalls and anti-patterns
+- Lets you maintain a **single source of truth** for all business logic
+- **Less boilerplate** so you can focus on the important parts of the app
+
+## Admin builders market overview
+
+The main problem with the tools currently in the Node.js and TypeScript ecosystem is that they require a major tradeoff between flexibility and speed. For a back office application developer, the backend and the database schema is an area of comfort. What's relatively challenging is building the UI and having a reasonable UX that can show the required data in a readable and editable format. And that's where all the admin builders come into the picture. If we closely look at the market, we can divide them into the following categories:
+
+### Fast development, less flexibility
+
+Most modern internal tool builders in the market fall into this category. Products like Retool, Appsmith act like a SaaS and provide a UI for rapid application development. The apps can be either hosted by you or them. They are excellent products for someone who's looking to build fast. The caveats are that you can't use your IDE, your existing codebase, or logic, and it can cost a lot as you scale.
```

Comment:
```suggestion
Most modern internal tool builders in the market fall into this category. Products like Retool, Appsmith act like a SaaS and provide a UI for rapid application development. The apps can either be hosted by you or them. They are excellent products for someone who's looking to build fast. The caveats are that you can't use your IDE, your existing codebase or custom scripts, and it can cost a lot as you scale.
```